tfexec/errors.go

```diff
@@ -19,6 +24,8 @@ func parseError(stderr string) error {
 	default:
 		return errors.New(stderr)
 	}
+
+	return fmt.Errorf("error from Terraform CLI:\n%s", stderr)
```

Comment:
This is unreachable since there is a `default` case.
```suggestion
```

Reply:
Thanks, I think that was a bad rebase!